io/zenoh-transport/tests/unicast_defragmentation.rs

```diff
@@ -204,3 +204,42 @@ fn transport_unicast_defragmentation_ws_only() {
         }
     });
 }
+
+#[cfg(feature = "transport_shm")]
+#[test]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.

Please apply the changes to `transport_ws` as well.